lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -541,9 +541,14 @@ void SPIRVEntryPoint::encode(spv_ostream &O) const {
 }
 
 void SPIRVEntryPoint::decode(std::istream &I) {
-  getDecoder(I) >> ExecModel >> Target >> Name >> Variables;
+  getDecoder(I) >> ExecModel >> Target >> Name;
+  // One fixed word is the name of the entry point. When we subtract FixedWC
```

Comment:
The name of the entry point can occupy more than one word. But for the `FixedWC` we assume that the name takes _at least_ one word. So your calculations are correct. The comment is just a bit confusing.